src/index.html

```diff
@@ -11,7 +11,201 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+    
+      <div class="main">
+        <fieldset class="field">
+          <legend>Personal information</legend>
+        
+          <label class="field__form">
+            Surname:
+
+            <input 
+            type="text" 
+            name="surname" 
+            required
+            >
```

Comment:
Fix indents for all inputs
```suggestion
            <input 
              type="text" 
              name="surname" 
              required
            >
```